### PR TITLE
Done HomeWork jv-lottery!

### PR DESCRIPTION
**Pull request by TkachukTara**

### CODE REVIEW

**Review by Rommelua — CHANGES_REQUESTED** [review]
image
Java Cl checks failed. Before making a Pull Request (or a new commit to an already existing PR), you need to write the mvn clean package command in the terminal and make sure that all checks pass. After that, the mentor will be able to review the homework

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lotteryObject = new Lottery();
+        Ball ball1 = lotteryObject.getRandomBall();
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/Application.java:6]: Use for loop for creating several objects of the same class

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        ListOfColors[] colors = ListOfColors.values();
+        int indexOfColor = new Random().nextInt(colors.length);
+        return colors[indexOfColor].toString();
+    }
+
+    public int getRandomINum() {
+        return new Random().nextInt(101);
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/ColorSupplier.java:13]: Don’t create several instances of the same class if you can use only one instance for your purposes

think about making a class-level field:
```
private Random random;
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery extends ColorSupplier {
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/Lottery.java:3]: Using inheritance here is a bad idea. 

Use composition here instead.

**Review by sokimaaa — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Application.java (hunk 1 above)
Comment (TkachukTara) [reply src/main/java/core/basesyntax/Application.java:6]: Done!

**Pull request closed without merging**